src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/TaskList/v2/TaskColumn.tsx

```diff
@@ -49,13 +49,17 @@ export const TaskColumn = ({
     setShowCreateTaskModal(false);
   };
 
+  const taskTitle =
+    status === 'IN_BACKLOG' ? 'New Backlog Task' : status === 'IN_PROGRESS' ? 'New In Progress Task' : 'New Done Task';
```

Comment:
Instead of checking if status is equal to 'IN_BACKLOG', you should do status === TaskStatus.IN_BACKLOG, and do the same for the other statuses